src/Gom/main.js

```diff
@@ -0,0 +1,159 @@
+import GObject from "gi://GObject";
+import Gom from "gi://Gom";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+import GLib from "gi://GLib";
+import Gio from "gi://Gio";
+
+const ItemClass = GObject.registerClass(
+  {
+    GTypeName: "Item",
+    Properties: {
+      id: GObject.ParamSpec.int(
+        "id",
+        "ID",
+        "An ID",
+        GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
+        0,
+        GLib.MAXINT32,
+        0,
+      ),
+      url: GObject.ParamSpec.string(
+        "url",
+        "URL",
+        "A URL",
+        GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
+        "",
+      ),
+    },
+  },
+  class ItemClass extends Gom.Resource { },
+);
+
+let adapter, repository;
+
+function initDatabase() {
+  adapter = new Gom.Adapter();
+  adapter.open_sync(":memory:");
```

Comment:
Should I use the demo directory as the path? Since it creates a bunch of other files the directory gets cluttered. I think a new folder would be a better option unless otherwise :)

Reply:
workbench crashes when I use a subdirectory and run the demo :/
hierarchy is like `Database -> sqlite -> sqlite.db`